app/src/main/res/values-es/strings.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <string name="app_name">MNML Screen Recorder</string>
+
+  <string name="overlay_permission_prompt">Se requieren permisos de superposición</string>
```

Comment:
On master, most of these have been moved to the strings file in the `engine` module 😄